src/Microsoft.Win32.SystemEvents/src/Microsoft/Win32/SystemEvents.cs

```diff
@@ -0,0 +1,1497 @@
+                    short atom = 0;
+
+                    //try to unregister it.
+                    if (0 != Interop.User32.UnregisterClass(WndClass.lpszClassName, Interop.Kernel32.GetModuleHandle(null)))
+                    {
+                        atom = Interop.User32.RegisterClass(WndClass);
+                    }
+
+                    if (atom == 0)
+                    {
+                        do
+                        {
+                            BumpQualifier();
+                            atom = Interop.User32.RegisterClass(WndClass);
+                        } while (atom == 0 && Marshal.GetLastWin32Error() == Interop.Errors.ERROR_CLASS_ALREADY_EXISTS);
+                    }
+                }
+            }
+
+            // And create an instance of the window.
+            //
+            IntPtr hwnd = Interop.User32.CreateWindowEx(
+                                                            0,
+                                                            WndClass.lpszClassName,
+                                                            WndClass.lpszClassName,
+                                                            Interop.User32.WS_POPUP,
+                                                            0, 0, 0, 0, IntPtr.Zero, IntPtr.Zero,
+                                                            hInstance, IntPtr.Zero);
+            return hwnd;
+        }
+
+        /// <internalonly/>
+        /// <devdoc>
+        ///    <para>Creates a new window timer asociated with the
+        ///       system events window.</para>
+        /// </devdoc>
+        public static IntPtr CreateTimer(int interval)
+        {
+            if (interval <= 0)
+            {
+                throw new ArgumentException(SR.Format(SR.InvalidLowBoundArgument, "interval", interval.ToString(System.Threading.Thread.CurrentThread.CurrentCulture), "0"));
+            }
+
+            EnsureSystemEvents(true, true);
+            IntPtr timerId = Interop.User32.SendMessage(new HandleRef(s_systemEvents, s_systemEvents._windowHandle),
+                                                        Interop.User32.WM_CREATETIMER, (IntPtr)interval, IntPtr.Zero);
+
+            if (timerId == IntPtr.Zero)
+            {
+                throw new ExternalException(SR.ErrorCreateTimer);
+            }
+            return timerId;
+        }
+
+        private void Dispose()
+        {
+            if (_windowHandle != IntPtr.Zero)
+            {
+                if (s_registeredSessionNotification)
+                {
+                    Interop.Wtsapi32.WTSUnRegisterSessionNotification(new HandleRef(s_systemEvents, s_systemEvents._windowHandle));
+                }
+
+                IntPtr handle = _windowHandle;
+                _windowHandle = IntPtr.Zero;
+
+                //we check IsWindow because Application may have rudely destroyed our broadcast window.
```

Comment:
Tiny nit, you could replace `//([^ ])` with `// \1`

Reply:
If we care, we should make the [codeformatter ](github.com/dotnet/codeformatter) do that.

Reply:
That url doesn't work - https://github.com/dotnet/codeformatter

Reply:
I cleaned things up a little, your regex isn't sufficient since it matches URLs and triple-slash comments.